did-pkh/did-pkh-method-draft.md

```diff
@@ -0,0 +1,214 @@
+}
+```
+## Operations
+
+### Create
+
+The blockchain account is validated according to [[caip]] and then appended to
+`did:pkh:{network}`, where `network` is the supported prefix corresponding to
+the blockchain where it is valid.
+
+### Read (Resolve)
+
+Resolution implements the following interface defined in [DID Core][]:
+```
+resolve(did, resolutionOptions) →
+   « didResolutionMetadata, didDocument, didDocumentMetadata »
+```
+
+Construct the DID Document for *did* as follows:
+- Parse the DID into its network id, *network* and account address, *address*,
+  according to Syntax and Interpretation above.
+- Initialize a DID document, *doc*, as a JSON-LD document, with `id` property
+  set to *did* and `@context` property set to `["https://www.w3.org/ns/did/v1",
+  "https://TODO.spruceid.com/TODO/v1"]`
+    - the contents of the latter context can be added manually if the above is
+      unavailable in the implementation process or cannot be made available at
+      runtime; see above in section [context](#context).
+- Construct the verification method ID, *vm*, by appending
+  "#blockchainAccountId" to *did*.
+- Construct the verification method object, *vmObj* as follows:
+  - define `id` as *vm* with `#blockchainAccountId` appended
+  - define `type` using the appropriate [[vM type]] value listed above per
+    blockchain/address type (see [networks](#networks) section)
+  - define `controller` as *vm*
+  - define `blockchainAccountId` as *vm* with `@` and the appropriate [[caip]]
+    chain id listed above per blockchain (see [networks](#networks) section)
+- Insert a property into *doc* with key name `verificationMethod` and a value of
+  an array containing only *vmObj*.
+- Insert a property into *doc* with key name `authentication` and a value of an
+  array containing only *vm*.
+- Insert a property into *doc* with key name `assertionMethod` and value of an
+  array containing only *vm*.
+- Construct a DID Resolution metadata object, *resMeta*.
+    - Set the `content-type` property of *resMeta* to "application/did+ld+json".
+- Construct a DID Document metadata object, *docMeta*.
+    - Set the `created` property of *docMeta* to the current time, in the format
+      according to [DID
+      Core](https://www.w3.org/TR/did-core/#did-document-metadata).
+- Return *resMeta*, *doc*, and *docMeta*.
+
+
+### Update
+
+No updates possible. did:pkh DID Documents are, like [did:key] documents, intended for local-only usage.
+
+### Delete
+
+No deletion possible. did:pkh DID Documents are, like [did:key] documents, intended for local-only usage.
+
+## Security & Privacy
+
+## Ref Impl
+
+|Author|name of implementation|link to pkh libraries|date registered|
+|:---:|:---:|:---:|:---:|
+|Spruce Systems, USA|[DIDKit](https://github.com/spruceid/didkit/)|`did-pkh` crate in ssi [core library](https://github.com/spruceid/ssi/tree/main/did-pkh)|July 2,2021|
+
+## Resources
+
```

Comment:
```suggestion

```
This looks like a mistake that the markdown below was thought to be a list of links. But they are actually markdown link definitions, referenced earlier in the page. So the section defined here is empty. Therefore I would suggest removing the section header.

More info about reference links in markdown:
- https://daringfireball.net/projects/markdown/syntax#link
- https://github.github.com/gfm/#reference-link

Reply:
Good catch! I actually just converted the links in the resources section to a list of footnote-style link definitions and forgot to remove the heading.